### PR TITLE
General cleanup

### PR DESCRIPTION
**Pull request by oesteban**

### CODE REVIEW

File: niworkflows/viz/validators.py [hunk 1]
```diff
@@ -18,7 +21,7 @@ def validate(self, css):
         stylesheet = self.parser.parse_stylesheet(css)
         for rule in stylesheet.rules:
             self.validate_no_fixed_position(rule)
-        if not stylesheet.errors is None and len(stylesheet.errors) > 0:
+        if stylesheet.errors is not None and len(stylesheet.errors) > 0:
```
Comment (effigies) [inline niworkflows/viz/validators.py]: This seems to be equivalent to:

```Python
if stylesheet.errors:
```

**Comment by effigies** [comment]
Good to merge?

**Comment by oesteban** [comment]
yess, kind of urgent actually. MRIQC is failing.

Have a look to the move I've made: mni from niworkflows.anat to
niworkflows.interfaces

**Comment by effigies** [comment]
Looks reasonable. Do you want to build MRIQC/FMRIPREP against this branch before merge? (I'm okay merging ASAP.)

**Comment by oesteban** [comment]
I know MRIQC works because I've been checking locally. I'd say we should go
ahead this time.

**Pull request merged**